templates/web/base/waste/bulky/intro.html

```diff
@@ -9,15 +9,18 @@ <h3>Before you start your booking</h3>
     <li>Requesting a [% service_name %] collection usually takes approximately <strong>10 minutes</strong></li>
   [% IF c.cobrand.moniker != 'brent' %]
     <li>You can request up to <strong>[% c.cobrand.bulky_items_maximum | numwords %] items per collection</strong></li>
-  [% ELSE %]
-    <li>You can arrange to have any of the following small items collected from outside your home for free, including textiles, small electricals, household batteries, and paint.
-    <li>Householders will be able to book up to three categories per collection. This could be, for example; 10 x AA batteries, two x 5L paint, one x black sack of textiles.</li>
-    <li>To see your options for coffee pod recycling please visit
-        <a href="https://www.podback.org/recycle-checker">https://www.podback.org/recycle-checker</a>
   [% END %]
     <li>Please ensure you have read the <strong><a href="[% c.cobrand.call_hook('bulky_tandc_link') %]" target="_blank">[% service_name %] collection</a></strong> page on the council’s website</li>
```

Comment:
Please make sure you have read the <strong>small items collection</strong> page on the council’s website.